### ISSUE TITLE
[syncfusion_flutter_charts] Dynamic  x-axis label

### ISSUE THREAD

**Issue by taosimple** [syncfusion/flutter-widgets#485]
In such a scenario: date time as x-axis, common label only show day, but the start of month show month+day. I tried `CategoryAxis` it not support repeat x label, `DateTimeAxis` or `DateTimeCategoryAxis` the `dateFormat` only support static format style, What should I do in this situation.

**Comment by SriramKiranSenthilkumar** [comment]
Hi @taosimple,

Greetings from Syncfusion. We have analyzed your query at our and we suspect that you are requesting for the axis label feature where for every first axis label of a month will display along with its month and so on. We have also attached a screenshot for your reference to check whether you are requesting for this improvement if yes then, we are glad to announce that this improvement for **DateTimeAxis** is already implemented at our end and this will get included in our upcoming Vol 4 main release which is expected to be rolled out by end of this month.
 
![image](https://user-images.githubusercontent.com/63577117/145996285-e87c50b3-3181-4159-ba7a-5bab129a00e7.png)


Regards,
Sriram Kiran
